docs/Reference/Reply.md

```diff
@@ -261,6 +261,7 @@ Sets the content type for the response. This is a shortcut for
 ```js
 reply.type('text/html')
 ```
+When set `Content-Type` with json type, if you don't set `charset`, it will use `utf-8` by default.
```

Comment:
Have to go into a meeting now but just stating that this doesn't read well.

Will review and update this comment when i'm out!

Reply:
Ooops, I missed this. I'll amend on main.